completions/perl

```diff
@@ -3,7 +3,7 @@
 _comp_cmd_perl__helper()
 {
     COMPREPLY=($(compgen -P "$prefix" -W \
-        "$("${2:-perl}" "${BASH_SOURCE[0]%/*}/../helpers/perl" "$1" "$cur")" \
+        "$("${1:-perl}" "${BASH_SOURCE[0]%/*}/../helpers/perl" "$2" "$cur")" \
         -- "$cur"))
     [[ $1 == functions ]] || _comp_ltrim_colon_completions "$prefix$cur"
```

Comment:
Hmm, I think this line should have also been updated as `[[ $2 == functions ]]`.

Reply:
@scop Can you include it in #988?

Reply:
Indeed, good catch, done in b85263a6de7f8e60f09489caac3f9a0d613e804f.